packages/scripts/README.md

```diff
@@ -382,6 +382,7 @@ This is how you execute the script with presented setup:
 This script automatically use the optimized config but sometimes you may want to specify some custom options:
 
 -   `--hot` – enables "Fast Refresh". The page will automatically reload if you make changes to the code. _For now, it requires that WordPress has the [`SCRIPT_DEBUG`](https://wordpress.org/documentation/article/debugging-in-wordpress/#script_debug) flag enabled and the [Gutenberg](https://wordpress.org/plugins/gutenberg/) plugin installed._
+-   `--no-watch` – ensures that the watcher will not monitor for changes and won't trigger tasks or builds automatically in response to those changes.
```

Comment:
This explanation may be a bit misleading to me since this option does not run the watcher itself. How about we just use [the description from the previously submitted PR](https://github.com/WordPress/gutenberg/pull/51707/files#diff-4a497b85ee79a156505d36fbe69b30773d32e1eeea45a2c2f9ff817328c22d18R390)?